packages/shared/src/components/auth/SocialRegistrationForm.tsx

```diff
@@ -174,6 +205,8 @@ export const SocialRegistrationForm = ({
           value={user?.name}
           valid={!nameHint && !hints?.name}
           hint={hints?.name || nameHint}
+          // value={name}
```

Comment:
This seems to be a mistake right?

Reply:
This was mean't to be removed, I have since deleted it